profile/README.md

```diff
@@ -0,0 +1,65 @@
+# Lombiq Technologies 
+
+
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the Orchard Core content management system. We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS.
+
+## Products:
+
+* [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips (FPGA). You can still update your software, but it will be a whole lot faster and more power-efficient.
+![Hastlayer-logo](https://user-images.githubusercontent.com/68704777/171685373-7a64e9bf-b0fc-431d-a8d1-9027e0989b15.png)
```

Comment:
When you insert an image, the part in square brackets should be human-readable for accessibility for people with visual impairment. Please do the same for the other images as well.

```suggestion
![Hastlayer logo](https://user-images.githubusercontent.com/68704777/171685373-7a64e9bf-b0fc-431d-a8d1-9027e0989b15.png)
```

Reply:
Please update this alt text for the other images as well. [Here](https://github.com/Lombiq/.github/pull/2/files#diff-0e83982cf6f4dabedcbf7b12029a56c6f4a728064ba99543fff8f227c0654b45R13), [here](https://github.com/Lombiq/.github/pull/2/files#diff-0e83982cf6f4dabedcbf7b12029a56c6f4a728064ba99543fff8f227c0654b45R33) and [here](https://github.com/Lombiq/.github/pull/2/files#diff-0e83982cf6f4dabedcbf7b12029a56c6f4a728064ba99543fff8f227c0654b45R41).